CHANGELOG.rst

```diff
@@ -0,0 +1,57 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Can you follow https://keepachangelog.com/ template instead ? jrl-cmakemodule `make release` command is able to deal with that